mkdocs.yml

```diff
@@ -52,84 +84,87 @@ extra:
-        - Apache Cassandra: storage-backend/cassandra.md
-        - Apache HBase: storage-backend/hbase.md
-        - Google Cloud Bigtable: storage-backend/bigtable.md
-        - Oracle Berkeley DB Java Edition: storage-backend/bdb.md
-        - InMemory Storage Backend: storage-backend/inmemorybackend.md
-    - Mixed Index Backends:
-        - Introduction: index-backend/index.md
-        - Search Predicates and Data Types: index-backend/search-predicates.md
-        - Index Parameters and Full-Text Search: index-backend/text-search.md
-        - Field Mapping: index-backend/field-mapping.md
-        - Direct Index Query: index-backend/direct-index-query.md
-        - Elasticsearch: index-backend/elasticsearch.md
-        - Apache Solr: index-backend/solr.md
-        - Apache Lucene: index-backend/lucene.md
-    - Advanced Topics:
-        - Advanced Schema: advanced-topics/advschema.md
-        - Eventually-Consistent Storage Backends: advanced-topics/eventual-consistency.md
-        - Failure & Recovery: advanced-topics/recovery.md
-        - Bulk Loading: advanced-topics/bulk-loading.md
-        - Graph Partitioning: advanced-topics/partitioning.md
-        - Datatype and Attribute Serializer Configuration: advanced-topics/serializer.md
-        - JanusGraph with TinkerPop’s Hadoop-Gremlin: advanced-topics/hadoop.md
-        - Monitoring JanusGraph: advanced-topics/monitoring.md
-        - Migrating from Titan: advanced-topics/migrating-titan.md
-        - Migrating from Thrift: advanced-topics/migrating-thrift.md
-        - JanusGraph Data model: advanced-topics/data-model.md
-        - JanusGraph Bus: advanced-topics/janusgraph-bus.md
+  - Introduction: index.md
+  - Getting Started:
+    - Installation: getting-started/installation.md
+    - Basic Usage: getting-started/basic-usage.md
+    - Gremlin Query Language: getting-started/gremlin.md
+    - Architectural Overview: getting-started/architecture.md
+  - Interactions with JanusGraph:
+    - Transactions: interactions/transactions.md
+    - Search Predicates and Data Types: interactions/search-predicates.md
     - Connecting to JanusGraph:
-        - Introduction: connecting/index.md
-        - Using Java: connecting/java.md
-        - Using Python: connecting/python.md
-        - Using .Net: connecting/dotnet.md
-    - Development: development.md
-    - Appendices: appendices.md
-    - Changelog: changelog.md
+      - Introduction: interactions/connecting/index.md
+      - Using Java: interactions/connecting/java.md
+      - Using Python: interactions/connecting/python.md
+      - Using .Net: interactions/connecting/dotnet.md
+  - Configuration:
+    - Introduction: configs/index.md
+    - Configuration Reference: configs/configuration-reference.md
+    - Config Example: configs/example-config.md
+  - Schema Management:
+    - Schema and Data Modeling: schema/index.md
+    - Advanced Schema: schema/advschema.md
+    - Index Management:
```

Comment:
I feel "Index Management" shall not be nested under "Schema Management".

Reply:
Aren't index in one way or other way of the schema?

I think the schema and index management of JanusGraph isn't straight forward.

Reply:
Fair enough. I just thought having "Index Management" deeply nested could create a delusion that indexing is not supported out of the box.